cla_backend/apps/call_centre/models.py

```diff
@@ -25,8 +25,16 @@ def save(self, *args, **kwargs):
         return obj
 
 
+class Organisation(TimeStampedModel):
+    name = models.CharField(max_length=255, blank=False, null=False, unique=True)
```

Comment:
I wouldn't include the `blank=False` and `null=False` to be honest. They're the defaults and, while it's not wrong to be explicit about them, at a glance one might assume they're being set to non-default values if they're being set (if I weren't code reviewing I think I might have made that mistake)